File: OOP's using JAVA/NestedClassDemoBook/src/Book.java

public class Book {
	private int bkid;
	private String bname;
	private String author;
	private Lesson l1;
	class Lesson{
		private int lessonid;
		private String lname;
		public void m1()
		{
			System.out.println("In lesson m1"+bkid);
		}
	}
	
	public Book() {
		super();
	}
	public Book(int bkid, String bname, String author) {
		super();
		this.bkid = bkid;
		this.bname = bname;
		this.author = author;
	}
	@Override
	public String toString() {
		return "Book [bkid=" + bkid + ", bname=" + bname + ", author=" + author + "]";
	}
}

File: .NET/Day11/BookMVCApp/BookQue/BookPortal/BOL/Book.cs
namespace BookPortal.BOL;
public class Book{
    public Book(){

    }
    public Book(int id,string Bname,string Author, int price)
    {
        this.Id = id;
        this.Bname = Bname;
        this.Author = Author;
        this.price = price;
    }
    
    public int Id{get;set;}
    public string Bname{get;set;}
    public string Author{get;set;}
    public int price{get;set;}

}